compiler/passes/convert-typed-uast.cpp

```diff
@@ -4050,7 +4068,8 @@ void TConverter::ActualConverter::convertActual(const FormalActual& fa) {
     INT_ASSERT(astActual);
 
     // Convert the actual and leave.
-    std::get<Expr*>(slot) = tc_->convertExpr(astActual, rv_);
+    auto actualExpr = tc_->convertExpr(astActual, rv_);
+    std::get<Expr*>(slot) = tc_->storeInTempIfNeeded(actualExpr, {});
```

Comment:
See the TODO, I think it would be good to have the callsites supply the types unless we're sure we're not going to accidentally invoke the old resolver.

So just get the type from `convertExpr` for now?